setup.py

```diff
@@ -16,7 +16,7 @@ def main():
         all_deps += extras[group_name]
     extras['all'] = all_deps
     extras['dev'] = [
-        'pylint==2.4.2',
+        'pylint==2.9.6',
```

Comment:
What do you think about `>=2.9.6`?

Reply:
When I ran `2.13.8` (current) on `neptune-client`, I got a lot of errors (pylint adds/changes validations between versions...) -- so right now I'd like to have a passing build + consistent version, and talk about upgrade later (in separate PR).